oidc_cli/oidc_impl/client/client.go

```diff
@@ -51,12 +52,7 @@ func NewClient(ctx context.Context, config *Config, clientOptions ...Option) (*C
 		ClientID:    config.ClientID,
 		RedirectURL: fmt.Sprintf("http://localhost:%d", server.GetBoundPort()),
 		Endpoint:    provider.Endpoint(),
-		Scopes: []string{
-			oidc.ScopeOpenID,
-			oidc.ScopeOfflineAccess,
-			"email",
-			"groups",
```

Comment:
Will we always have to add all these scopes manually now? If so, would it be worth adding a function `AddDefaultScopes` that adds all of these?

Reply:
Yeah like the `AddDefaultScopes` you defined here! Adding default scopes if none are provided would be nice too if possible, that way it's backwards compatible